config/default.yml

```diff
@@ -1,11 +1,15 @@
-jwt:
-  expiresIn: '1y' # default token expire time 1 year. Examples - https://github.com/vercel/ms#examples
+accounts:
+  tokenExpiry: '1y' # default token expire time 1 year. Examples - https://github.com/vercel/ms#examples
```

Comment:
imo, access token expiries should be short say a day etc. This way even if it falls in the wrong hands it can't do much damage.

Reply:
Addressed - Changed expiry to `1d`